server/CLI/manager.py

```diff
@@ -0,0 +1,205 @@
+#!/usr/bin/python3
+# Copyright 2019 Anton Maksimovich <antonio.maksimovich@gmail.com>
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+"""Utility for controlling(creating/deleting/printing) the clients instances"""
+
+import asyncio
+import os
+import secrets
+import ssl
+import string
+import sys
+from argparse import ArgumentParser
+from email.message import EmailMessage
+from smtplib import (SMTP_SSL, SMTPHeloError, SMTPAuthenticationError,
+                     SMTPException)
+from asyncpg.exceptions import UniqueViolationError
+from dotenv import load_dotenv
+from huntflow_reloaded.models import User, gino_run  # pylint: disable=import-error
+
+
+load_dotenv()
+
+
+def send_mail(smtp_server, port, sender_email, sender_password, receiver_email,  # pylint: disable=redefined-outer-name,too-many-arguments
+              message):  # pylint: disable=redefined-outer-name
+    """Sending credentials to user. """
+    context = ssl.create_default_context()
+    with SMTP_SSL(smtp_server, port, context=context) as server:
+        server.login(sender_email, sender_password)
+        msg = EmailMessage()
+        msg['Subject'] = 'Your credentials'
+        msg['From'] = sender_email
+        msg['To'] = receiver_email
+        msg.set_content(message)
+        server.sendmail(sender_email, receiver_email, msg.as_string())
+
+
+def generate_password(pass_len):
+    """Creating a password of the specified length. """
+    chars_set = string.ascii_letters + string.digits
+    return ''.join(secrets.choice(chars_set) for _ in range(pass_len))
+
+
+async def create_user(email, password):  # pylint: disable=redefined-outer-name
+    """Creating the user instance. """
+    record = await User.create(
+        email=email,
+        password=password
+    )
+    return record
+
+
+async def delete_user(email):
+    """Deleting the user instance. """
+    user = await User.query.where(User.email == email).gino.all()
+    if not user:
+        return False
+    user = user[0]
+    await user.delete()
+    return True
+
+
+async def list_of_users():
+    """Printing the list of users. """
+    all_users = await User.query.gino.all()
+    return all_users
+
+
+def connect_to_postgres(loop, postgres_host, postgres_port, postgres_user,
+                        postgres_password):
+    """Connecting to postgres server. """
+    if not all([postgres_host, postgres_user, postgres_password,
+                postgres_port]):
```

Comment:
You can check only `postgres_password` because you have default values for others. Also, you can check if it exists during parsing the command line args via `parser.error`.